src/Helper/CreatesIiifUri.php

```diff
@@ -20,12 +20,18 @@
         if ($width && $height) {
             if (($width / $height) !== ($image->getWidth() / $image->getHeight())) {
                 $region = IiifRegionCalculator::calculateForImage($image, $width, $height);
-                if ($width === $image->getWidth()) {
-                    $width = '';
-                }
                 if ($height === $image->getHeight()) {
                     $height = '';
+                } elseif ($width && $height === (int) ($width * ($image->getHeight() / $image->getWidth()))) {
+                    $height = '';
+                }
+
+                if ($width === $image->getWidth()) {
+                    $width = '';
+                } elseif ($height && $width === (int) ($height * ($image->getWidth() / $image->getHeight()))) {
+                    $width = '';
```

Comment:
Prefer a size of `2000,` over `2000,1333` when the resulting height is `1333`.

Reply:
This looks like one of those rare `// comments` that do not duplicate information already in the code

Reply:
Can extract `$this->widthNotNecessary($image, $width, $height)`/`$this->heightNotNecessary($image, $width, $height)` covering both the conditions of each `if/elseif` block